stdlib/REPL/test/lineedit.jl

```diff
@@ -375,6 +375,16 @@ let buf = IOBuffer()
     @test content(buf) == "βγαεδ"
     LineEdit.edit_transpose_chars(buf)
     @test content(buf) == "βγαδε"
+
+    seek(buf, 0)
+    @inferred(LineEdit.edit_clear(buf))
```

Comment:
And a quick question: this probably doesn't need to have a `@inferred`, right? (Though I imagine it doesn't really matter.)

Reply:
I think it was added to the other places as some type instabilities in the REPL was addressed.